src/views/HomePage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Rename views to pages and export homepage as regular not default, then export it from the index file in the pages folder